### ISSUE TITLE
Should Molpy Down remove dragons?

### ISSUE THREAD

**Issue by TheSnacker** [eternaldensity/Sandcastle-Builder#1306]
I have added an update that will delete dragons when you Coma (in theory), and I just wanted to get an option on if it should reset on Molpy Down.  I would think the answer would be yes, but I just wanted to see what others think before I add that.

**Comment by EPSIL0N** [comment]
I guess it really depends what is decided with regards to the other long haul things as currently I don't think I would molpy down now for fear of losing certain things and probably wouldn't want to MD if it cost all my dragons.

**Comment by eternaldensity** [comment]
This reminds me, I was going to make a Dragon Overlord which protects (the older Boost-based) dragons from Molpy Down but never got around to even writing the idea down.

Maybe that could be Dragon Lord and then Dragon Overlord lets you protect (some?) bred dragons too.

**Comment by erp-lsf** [comment]
Maybe save(on that later) them, but make them inactive, like, 'dazed' for some time after Molpy Down?

**Comment by eternaldensity** [comment]
Now that's a clever idea, I like it!

**Comment by pickten** [comment]
Maybe a period of time equal to hatch time+mature time for each group? Feels MD-ish, but not overly so.

**Comment by Calamitizer** [comment]
I think it would be satisfying to have some system of "awakening" the dragon's would be more satisfying. Maybe "From Sleep Awake": boops100 awake 100 dragons at the cost of infinite something, with no repeats (like WIWTMP). They awake in order from 0-99, 100-199, etc. Or some boost like that. Then eventually "Bag of Molting": All dragons stay awake through MD, and multibuy dragon Boosts (like armor) stay bought.

**Comment by erp-lsf** [comment]
Any new "Bag" would only slog current prize system even more, and move it further from redesign.

**Comment by Calamitizer** [comment]
Oh, are we looking for a reboot of the MD system, not just an extension?

**Comment by eternaldensity** [comment]
Yeah that's been marked as fairly terrible and needing revamping for a long
time.

**Comment by Calamitizer** [comment]
Okay! I agree it is not so great. I didn't realize that was a shared opinion, though.

**Comment by EPSIL0N** [comment]
It is not very well liked but there is a lot of silence on the issue as most people seem to just binge and get all the prizes or find themselves at a point in time where it is absolutely insane to start getting the prizes so late so they never start.

I've been meaning to do a test MD lately just to see what is lost and how things work.

**Comment by pickten** [comment]
MDing is a pain, because you basically rush to get NP bots for Factory automation, then build all the mould boosts (at 17/NP, which is too damn slow IMO), then get lots of AC, get back up to decent levels of everything, and start over.

**Comment by erp-lsf** [comment]
Is there reasonable way to fix the bicycle?

**Comment by eternaldensity** [comment]
> Is there reasonable way to fix the bicycle?

Pardon?

**Comment by erp-lsf** [comment]
Ahah, sorry :smile: 
I was trying to ask is there a way to revamp the Bag system to make it less painful, or just better start from scratch.

**Comment by eternaldensity** [comment]
Do you mean the Prize system?

**Comment by erp-lsf** [comment]
Yes.

**Comment by EPSIL0N** [comment]
I think from past discussion we are probably looking at a full rework as I believe the longer a user is past the ideal MD time the worse it gets to do a MD.

**Comment by pickten** [comment]
So, when I mentioned 75 boosts a few days ago, almost all were Prizes, because I'm trying to redo Prizes.

The new system will be a little different, but not overly so. Basically, 8 tiers, each tier gives 2^128 more bag of holding space, and makes some boosts be maintained (so eventual goal is free MD at tier 8, at which point we might add an intermediate between MD and Coma, but IDK). Bags give a variety of points of varying tiers. If you have any points, you unlock the "Casino" which lets you trade in points for random Prizes. Furthermore, each Tier is themed after, and is intended to be completed during, a different stage of SB progression. 

So far, I've got (first two need rework b/c they more or less require players to have inf of each already):

Sand/Castles (needs rework)

Glass (needs rework - a bit of a larger tier than the others)

Prints/Crystals (done - a bit of a larger tier than the others)

Weird - shards/panes, ritual stuff, goats, and a bit of draconic (mostly done)

Mustard/Goats/draconic (mostly done - draconic aspect is more connected
to a Prize I've temporarily called THOOM that prevents RK attacks for a while)

??? (not started)

Bonemeal (not started)

??? (not started - special)

With a few non-MD ways to get prizes, it's set up so that those who wish to avoid MD will be able to finish anyways, but more slowly. So... yeah. 70-ish prizes. Around 40 of which are written (not coded). And 10 or so Bags, and 10 or so Stuffs (the points). Also, the sheer size was the big reason I wanted to avoid explaining this yet. It's coming together though. Slowly.

Keeping in mind that it's not really finalized, probably pretty unbalanced, may not be finished for a while, and that names/descriptions/stats are probably going to be changed after I'm done drafting (which is partly why I explain almost all of the references I made, made *much* more than I normally might, and made more obvious ones as well), here's a link to the google doc I've been using, moved to a gist (warning: as a result, some italics/bolding got screwed up, but I don't feel like fixing that right now):

https://gist.github.com/pickten/1b42262df2a6a2821f4c

**Comment by eternaldensity** [comment]
I think holding space increases from better Bags (ah now i get why the word
Bags was used before, because some of the actually important Prizes are the
ones that are Bag of Something) should be multiplicative rather than
additive.  And maybe there needs to be some per-stuff calibration.  Cos
just adding some fixed amount means that some increases will be
significantly more meaningful than others.

**Comment by pickten** [comment]
The multiplicative carrying capacity was intended. Sorry if that wasn't clear. That's how MD was supposed to approach being free (which is my goal by the end of T8, so that production can go crazy, which would ideally be the next step). Probably does need to be some calibration, but IDK, since Infinite carrying should work the same for all stuffs and that is the objective.

**Comment by eternaldensity** [comment]
I don't bother with the mould boosts on MD (they're not in the queue if you
don;t into glass production) and glass saw soon gives infinite blocks for
Baf of Holding

**Comment by pickten** [comment]
Oh, smart. I've always done it the obvious way. I'll take that into consideration when re-reworking the first couple tiers.

**Comment by eternaldensity** [comment]
Oops this time I forgot to get infinite tools first, so now getting back is gonna take a little longer.  Oh well.
